test/unit/specs/components/AudioTrack/v-audio-track.spec.js

```diff
@@ -88,9 +91,6 @@ describe('AudioTrack', () => {
     options = {
       propsData: props,
       stubs,
```

Comment:
The `$nuxt` mock here overrides the mock in `test/unit/setup-after-env.js`, which is necessary to use Sentry.